src/main/export/__tests__/secondToTimestamp.test.ts

```diff
@@ -1,10 +1,10 @@
-import { secondToTimestamp } from '../../timeUtils';
+import { secondToEDLTimestamp } from '../../timeUtils';
```

Comment:
nitpick: acronyms in variable and function names should be treated as words i.e. `secondToEdlTimestamp`